bokeh/themes/theme.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import absolute_import, print_function
+
+import yaml
+
+from ..properties import HasProps
+
+# whenever we cache that there's nothing themed for a class, we
+# use this same dict instance, so we don't have a zillion empty
+# dicts in our caches.
+_empty_dict = dict()
+
+class Theme(object):
```

Comment:
have been thinking about that too. It might be confusing if it's a model before we really use it as one, so maybe better to wait and do it when/if we decide to send it to the client ?